src/main/java/kr/allcll/checkll/api/AdminController.java

```diff
@@ -13,21 +15,21 @@ public AdminController(AdminService adminService) {
         this.adminService = adminService;
     }
 
-    @GetMapping("/api/setinfo")
-    public ResponseEntity<Void> setUpInfo() {
-        adminService.setUpInfoByString(""); //수동으로 넣나
+    @PostMapping("/api/set-credential")
+    public ResponseEntity<Void> setCredential(@RequestBody SetCredentialRequest request) {
+        adminService.setUpCredential(request.credential());
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping("/api/start-session")
+    @PostMapping("/api/start-session")
     public ResponseEntity<Void> startSession() {
-        adminService.startInitialSession();
+        adminService.setUpSession();
         return ResponseEntity.ok().build();
     }
 
     @GetMapping("api/end-session")
     public ResponseEntity<Void> endSession() {
-        adminService.endUpSession();
+            adminService.shutDownSession();
```

Comment:
C: 여기 공백이 있으면 안되는 것 맞나요? 

```suggestion
        adminService.shutDownSession();
```

Reply:
엥 말도안돼 분명 정렬을했는데 무슨일일까요